README.md

```diff
@@ -97,15 +97,25 @@ Have questions, comments or feedback? [Join our discord](https://discord.gg/rhsy
 
 You can find tldraw on npm [here](https://www.npmjs.com/package/@tldraw/tldraw?activeTab=versions).
 
-At the moment the tldraw package is in alpha. We also ship a canary version which is always up to date with the main branch of this repo.
+## Contribution
 
-## License
+Please see our [contributing guide](https://github.com/tldraw/tldraw/blob/main/CONTRIBUTING.md). Found a bug? Please [submit an issue](https://github.com/tldraw/tldraw/issues/new).
 
-The source code for various apps and packages in this repository (as well as our 2.0+ distributions and releases) are currently licensed under Apache-2.0. These licenses are subject to change in our upcoming 2.0 release. If you are planning to use tldraw in a commercial product, please reach out at [hello@tldraw.com](mailto://hello@tldraw.com).
+## Open source license
 
-## Contribution
+tldraw is open source under the GNU Affero General Public License Version 3 (AGPLv3) or any later version. You can find it [here](https://github.com/tldraw/tldraw/blob/master/LICENSE.md). All packages are distributed under the same license.
 
-Please see our [contributing guide](https://github.com/tldraw/tldraw/blob/main/CONTRIBUTING.md). Found a bug? Please [submit an issue](https://github.com/tldraw/tldraw/issues/new).
+The GNU Affero General Public License is a free, copyleft license for software and other kinds of works, specifically designed to ensure cooperation with the community in the case of network server software. The AGPL-3.0 license allows users to use and modify the software as long as they keep it open source and provide any modifications or derivative works under the same license.
+
+## Commercial license
+
+If you wish to use this project in closed-source software or otherwise do not want to comply with the AGPL-3.0, you need to purchase a commercial license. Please contact us at [hello@tldraw.com](mailto:hello@tldraw.com) for more information about obtaining a commercial license.
+
+The dual licensing model supports the development of the project by providing an open source option for those who are contributing back to the community, while also supporting commercial usage for entities that are not able to do so.
```

Comment:
I think this stuff needs to be much clearer to aid adoption. I'd suggest having just one section called 'Licensing' that gets to the point. Something like

```md
## Licensing

tldraw's source code is published under the restrictive AGPL-3.0 license, which means you must obtain a separate license from us if you wish to use tldraw in a commercial project or any project not published under AGPL-3.0.

Obtain a license by contacting us at [hello@tldraw.com](mailto:hello@tldraw.com). Let us know what you're working on and how tldraw will fit in.

We are quite happy to grant free licenses on a case-by-case basis to people using tldraw for charitable projects, research projects, and non-AGPL open source projects.
```

Reply:
Good feedback, I've tightened it up.